lib/rules/no-restricted-globals.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @fileoverview Restrict usage of specified globals.
+ * @author Benoît Zugmeyer
```

Comment:
Add copyright and license info http://eslint.org/docs/developer-guide/code-conventions.html#file-format